src/differentials/combinations.jl

```diff
@@ -0,0 +1 @@
+Base.setindex!(z::AbstractZero, ::AbstractThunk, ::Any) = z
```

Comment:
I do not understand what this does

Reply:
deleted, I don't think it makes sense. I was in the automatic mode of "everything called on zero is zero" but it's completely wrong here, thanks.